frontends/mit-open/src/pages/HomePage/TestimonialsSection.tsx

```diff
@@ -100,19 +108,24 @@ const TestimonialCardQuote = styled.div({
   "div.testimonial-quote-opener": {
     color: theme.custom.colors.mitRed,
     fontStyle: "normal",
-    height: pxToRem(80),
+    fontWeight: theme.typography.fontWeightBold,
```

Comment:
In `typography.ts`, we define 3 font weights that match the designs:

```ts
const fontWeights = {
  text: {
    roman: 400,
    medium: 500,
    bold: 700,
  },
}
```

Thus, the values you end up getting here come from the MUI defaults. In `typography.ts`, can we assign the MUI properties to our values specified here? For example:

```ts
const globalSettings: ThemeOptions["typography"] = {
  // Note: Figma calls this "Neue Haas Grotesk Text", but that is incorrect based on Adobe's font family.
  fontFamily: "neue-haas-grotesk-text, sans-serif",
  fontWeightLight: fontWeights.text.roman,
  fontWeightRegular: fontWeights.text.roman,
  fontWeightMedium: fontWeights.text.medium,
  fontWeightBold: fontWeights.text.bold,
  ...
```

The MUI defaults don't end up being functionally different (MUI has 300 for light and we use 400) but using the values defined here will ensure that the values match the font we've chosen, in this case `neue-haas-grotesk-text, sans-serif`.

We actually have a linting rule that is supposed to disable manually specifying `font-weight`, but I think it was allowed here because it only disallows numeric weights. Anyway, this is definitely a special case as it's not really regular text and is effectively being used here as an icon. Maybe this whole thing would be better as an SVG, but that's another conversation.

Reply:
I basically just added the font weights into the theme options as specified. There are a couple other places where this gets used:
- `mit-open/src/pages/HomePage/NewsEventsSection.tsx`
- `ol-components/src/components/ChoiceBox/ChoiceBox.tsx`

The onboarding is broken for me locally at the moment, which is where ChoiceBox seems to be used exclusively, but the news and events section didn't seem to change.

Reply:
My point was more that the MUI font weight properties (i.e. `theme.typography.fontWeightbold`) are not currently customized by our theme, so when you call those properties now you are going to get the MUI defaults. For `theme.typography.fontWeightLight` for example, you get 300 back whereas in our case that should be 400. This recommendation is based on a conversation with @ChristopherChudzicki about how our specified font weights relate to the imported Abobe font.